experimental/iterators/test/python/dialects/iterators/dialect.py

```diff
@@ -98,3 +103,33 @@ def testEndToEnd():
   engine = ExecutionEngine(mod)
   # CHECK: (6)
   engine.invoke('main')
+
+
+@run
+# CHECK-LABEL: TEST: testEndToEndWithInput
+def testEndToEndWithInput():
+  mod = Module.parse('''
+      !struct_type = !llvm.struct<(i32,i64)>
+      func.func @main(%input: !tabular.tabular_view<i32,i64>)
+          attributes { llvm.emit_c_interface } {
+        %stream = "iterators.tabular_view_to_stream"(%input)
```

Comment:
can we use nicer assembly everywhere plz?

Reply:
Applied in latest commit.